assets/src/blocks/ActionsList/ActionsListSidebar.js

```diff
@@ -0,0 +1,41 @@
+import {InspectorControls} from '@wordpress/block-editor';
+import {TextControl, PanelBody} from '@wordpress/components';
+import {BLOCK_NAME} from './ActionsList';
+
+const {addFilter} = wp.hooks;
+const {__} = wp.i18n;
+
+// Customize the ActionsList sidebar options.
+export const registerActionsListSidebar = () => {
+  const withActionsListOptions = BlockEdit => props => {
+    const {attributes, name, setAttributes} = props;
+    const isActionsList = name === 'core/query' && attributes.namespace === BLOCK_NAME;
+
+    return isActionsList ? (
+      <>
+        <InspectorControls>
+          <PanelBody title={__('Planet 4 Options')}>
+            <TextControl
+              label={__('Actions count', 'planet4-blocks-backend')}
```

Comment:
I think we can skip that completely. If we document the block properly we can guide editors to use the toolbar section to adjust the number of Actions to show.